routes/web.php

```diff
@@ -62,6 +64,18 @@
 
 Route::get('/project/{id}/testmeasurements', 'ManageMeasurementsController@show');
 
+/* TODO: Figure this forbidden response
+
+Route::get('/register', function (Request $request) {
```

Comment:
@williamjallen, this was my attempt at re-routing.  If you have another example of doing this, I'd like to see it.  Thanks!!

Reply:
You might have some luck with adding a check in this function:
https://github.com/Kitware/CDash/blob/50192574115056ec97560530e7e709533a125300/app/Http/Controllers/Auth/RegisterController.php#L48-L59

You already have a similar check here:
https://github.com/Kitware/CDash/blob/50192574115056ec97560530e7e709533a125300/app/Http/Controllers/Auth/RegisterController.php#L133-L135

It is unclear to me why we need two different functions which show the registration page.  Perhaps a good refactor here would be to have `register()` call `showRegistrationForm()` to eliminate the code duplication here.